partner_financial_risk/README.rst

```diff
@@ -0,0 +1,67 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+======================
+Partner Financial Risk
+======================
+
+Adds a new page in partner to manage the partner's *Financial Risk*
+If any limit is exceed the the partner is blocked to confirm sale orders.
```

Comment:
@rafaelbn thanks for the explanation. My question wider broader than that: there are multiple fields and logic in the code that is not crystal clear and that might require some more explanation.
Not a real blocking point but then the module is never really used if you cannot really understand it (which was my case this morning)